use-cases/vehicle-routing/src/test/java/org/acme/vehiclerouting/rest/VehicleRoutePlanResourceTest.java

```diff
@@ -22,6 +22,40 @@ public class VehicleRoutePlanResourceTest {
 
     @Test
     public void solveDemoDataUntilFeasible() {
+        VehicleRoutePlan solution = solveDemoData();
+        assertTrue(solution.getScore().isFeasible());
+    }
+
+    @Test
+    void analyze() {
+        VehicleRoutePlan solution = solveDemoData();
+        assertTrue(solution.getScore().isFeasible());
+
+        String analysis = given()
+                .contentType(ContentType.JSON)
+                .body(solution)
+                .expect().contentType(ContentType.JSON)
+                .when()
+                .put("/route-plans/analyze")
+                .then()
+                .extract()
+                .asString();
+        assertNotNull(analysis); // Too long to validate in its entirety.
```

Comment:
Can we at least check the difference between shallow and full result?

Reply:
The same as above.